### DISCUSSION TITLE
Support for pairing ZigBee devices with Installation Code

### DISCUSSION THREAD

**Discussion by maxx2097** [arendst/Tasmota#21342] [Ideas]
Some devices (e.g. Bosch BTH-RA radiator thermostat) must be pre-registered on a bridge device using an Installation Code (usually printed as a QR code on the device) to be able to pair it.
Are there any plans to add this feature to the Tasmota ZigBee implementation?

**Comment by Jason2866** [comment]
Will not be done from Tasmota development team since it is manufacturer dependent.

**Comment by s-hadinger** [comment]
No plans for this. We are investing on Matter which offer such mechanism with QRcode and has more future potential
